Bench/BenchViewController.swift

```diff
@@ -1,108 +0,0 @@
-import UIKit
-import MapboxDirections
-import MapboxNavigation
-import MapboxCoreNavigation
-import TestHelper
```

Comment:
Usage of this dependency makes successful compilation impossible at this point. If we'd like to keep using these targets we'll have to find a way to be able to use `TestHelper` not only in `MapboxNavigation.xcodeproj` or implement helper method in `Bench` target itself.